index.html

```diff
@@ -207,54 +207,12 @@ <h3 class="section-subheading text-muted">Library of awesome works by IEDC</h3>
                             <img class="img-fluid" src="assets/img/portfolio/03-thumbnail.jpg" alt="..." />
                         </a>
                         <div class="portfolio-caption">
-                            <div class="portfolio-caption-heading">Finish</div>
-                            <div class="portfolio-caption-subheading text-muted">MVPs</div>
+                            <div class="portfolio-caption-heading">IEDC Summit</div>
```

Comment:
![image](https://user-images.githubusercontent.com/38415384/123513094-041a1000-d640-11eb-913e-455de6dc8042.png)

A dangling `<li>` tag, in the popup, pretty easy fix..

Reply:
oh is didn't see that there my bad

Reply:
u can close this, I could do another request.